lib/Views/src/layouts/DateTimeView64x64.h

```diff
@@ -60,31 +61,28 @@
 /**
  * View for 64x64 LED matrix with date and time.
  */
-class DateTimeView64x64 : public IDateTimeView
+class DateTimeView64x64 : public DateTimeViewGeneric
 {
 public:
 
     /**
      * Construct the view.
      */
     DateTimeView64x64() :
-        IDateTimeView(),
-        m_fontType(Fonts::FONT_TYPE_DEFAULT),
-        m_textWidget(TEXT_WIDTH, TEXT_HEIGHT, TEXT_X, TEXT_Y),
-        m_lampWidgets{{LAMP_WIDTH, LAMP_HEIGHT, LAMP_0_X , LAMP_Y},
-                      {LAMP_WIDTH, LAMP_HEIGHT, LAMP_1_X , LAMP_Y},
-                      {LAMP_WIDTH, LAMP_HEIGHT, LAMP_2_X , LAMP_Y},
-                      {LAMP_WIDTH, LAMP_HEIGHT, LAMP_3_X , LAMP_Y},
-                      {LAMP_WIDTH, LAMP_HEIGHT, LAMP_4_X , LAMP_Y},
-                      {LAMP_WIDTH, LAMP_HEIGHT, LAMP_5_X , LAMP_Y},
-                      {LAMP_WIDTH, LAMP_HEIGHT, LAMP_6_X , LAMP_Y}},
-        m_dayOnColor(DAY_ON_COLOR),
-        m_dayOffColor(DAY_OFF_COLOR)
+        DateTimeViewGeneric(),
+        m_lastUpdateSecondVal(-1)
     {
         /* Disable fade effect in case the user required to show seconds,
          * which will continuously trigger the fading effect.
          */
         m_textWidget.disableFadeEffect();
+
+        /*
+         * Move digital clock to lower part of analog clock.
+         * Analog clock is also shifted in X by one as the
+         * mid point is 32.
+         */
+        m_textWidget.move(1, 47);

```

Comment:
Did you consider that the font size may change? It can be influenced via the slot configuration.